Makefile

```diff
@@ -93,14 +93,14 @@ build-rocksdb: $(LIB_ROCKSDB)
 
 $(BROKER_BIN): $(LIB_ROCKSDB)
 ifdef linux-os-host
-	CGO_ENABLED=1 CGO_CFLAGS="-I$(ROCKSDB_INCLUDE_DIR)" \
-	CGO_LDFLAGS="-L$(ROCKSDB_LIB_DIR)" \
+	CGO_ENABLED=1 CGO_CFLAGS="${CGO_CFLAGS}-I$(ROCKSDB_INCLUDE_DIR)" \
+	CGO_LDFLAGS="${CGO_LDFLAGS}-L$(ROCKSDB_LIB_DIR)" \
```

Comment:
Space required between `${CGO_LDFLAGS}` and `-L$(ROCKSDB_LIB_DIR)`

Reply:
ditto.